pgbouncer-mixin/dashboards.libsonnet

```diff
@@ -47,25 +47,25 @@ local logslib = import 'logs-lib/logs/main.libsonnet';
         + root.applyCommon(vars.multiInstance, uid + '-overview', tags, links { pgbouncerOverview+:: {} }, annotations, timezone, refresh, period),
     }
     +
-    if this.config.enableLokiLogs
-    then
+    if this.config.enableLokiLogs then
       {
         logs:
-          logslib.new(prefix + 'PgBouncer logs',
-                      datasourceName=this.grafana.variables.datasources.loki.name,
-                      datasourceRegex=this.grafana.variables.datasources.loki.regex,
-                      filterSelector=this.config.filteringSelector,
-                      labels=this.config.groupLabels + this.config.instanceLabels + this.config.extraLogLabels,
-                      formatParser=null,
-                      showLogsVolume=this.config.showLogsVolume,
-                      logsVolumeGroupBy=this.config.logsVolumeGroupBy,
-                      extraFilters=this.config.logsExtraFilters)
+          logslib.new(
+            prefix + 'PgBouncer logs',
+            datasourceName=this.grafana.variables.datasources.loki.name,
+            datasourceRegex=this.grafana.variables.datasources.loki.regex,
+            filterSelector=this.config.filteringSelector,
+            labels=this.config.groupLabels + this.config.extraLogLabels,
```

Comment:
i notice there's a variable/selector for "Database" that gets created. Does this make sense?

Reply:
Not really, changed it to include instance instead